package/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,22 @@
+﻿
+$ErrorActionPreference = 'Stop';
+
+$packageName = 'punktf'
+$destDir     = Join-Path $(Get-ToolsLocation) $packageName
+$url64       = 'https://github.com/neovim/neovim/releases/download/release/punktf.exe'
+
+
+$packageArgs = @{
+    packageName   = $packageName
+    installerType = 'exe'
+    url64bit      = $url64
+
+    softwareName  = 'punktf*'
+
+    checksum64    = ''

```

Comment:
I will make a release so you can fill this

Reply:
Please add the checksums to the release description

Reply:
Will do